### PR TITLE
[FEATURE] Add Rigid ‘get_dofs_frictionloss’ public API method.

### PR DESCRIPTION
**Pull request by ax-anoop**
This commit adds the missing getter method for DOF friction loss parameter, which was previously only available via a setter. This completes the API consistency with other DOF parameters like damping, armature, and stiffness.

Changes:
- Added get_dofs_frictionloss() to RigidSolver (solver level)
- Added get_dofs_frictionloss() to RigidEntity (entity level)
- Follows the same pattern as existing get_dofs_damping() method

```py
import genesis as gs
gs.init(backend=gs.cpu)

scene = gs.Scene(show_viewer=True)
plane = scene.add_entity(gs.morphs.Plane())
franka = scene.add_entity(
    gs.morphs.MJCF(file='xml/franka_emika_panda/panda.xml'),
)

scene.build()

#for i in range(1000):
#    scene.step()

print(franka.get_dofs_kp())
print(franka.get_dofs_frictionloss())
franka.set_dofs_frictionloss([1,1,1,1,1,1,0.1,0.1,0.1])
print(franka.get_dofs_frictionloss())
```

## Related Issue

Resolves Genesis-Embodied-AI/Genesis#<your-issue-number>

## Checklist:

- [ ] I read the **CONTRIBUTING** document.
- [ ] I followed the `Submitting Code Changes` section of **CONTRIBUTING** document.
- [ ] I tagged the title correctly (including BUG FIX/FEATURE/MISC/BREAKING)
- [ ] I updated the [documentation](https://github.com/Genesis-Embodied-AI/genesis-doc) accordingly or no change is needed.
- [ ] I tested my changes and added instructions on how to test it for reviewers.

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by duburcqa** [comment]
Thank you for your contribution. Please update the unit tests `test_set_dofs_frictionloss_physics` and `test_data_accessor` accordingly.

**Comment by ax-anoop** [comment]
> Thank you for your contribution. Please update the unit tests `test_set_dofs_frictionloss_physics` and `test_data_accessor` accordingly.

done, let me know if its ok

File: tests/test_rigid_physics.py [hunk 1]
```diff
@@ -1692,12 +1692,16 @@ def test_set_dofs_frictionloss_physics(gs_sim, tol):
...
+    frictionloss = robot.get_dofs_frictionloss()
+    np.testing.assert_allclose(frictionloss, np.array([0.0, 0.0]), atol=tol)
     for _ in range(10):
         gs_sim.step()
     velocity_zero = gs_sim.rigid_solver.dofs_state.vel.to_numpy()[:, 0]
 
     robot.set_dofs_velocity(initial_velocity)
     robot.set_dofs_frictionloss(np.array([1.0, 0.0]))
+    frictionloss = robot.get_dofs_frictionloss()
+    np.testing.assert_allclose(frictionloss, np.array([1.0, 0.0]), atol=tol)
```
Comment (duburcqa) [inline tests/test_rigid_physics.py]: You must use our custom `assert_allclose`

File: tests/test_rigid_physics.py [hunk 2]
```diff
@@ -1707,6 +1711,8 @@ def test_set_dofs_frictionloss_physics(gs_sim, tol):
 
     robot.set_dofs_velocity(initial_velocity)
     robot.set_dofs_frictionloss(np.array([0.5]), dofs_idx_local=[0])
+    frictionloss = robot.get_dofs_frictionloss(dofs_idx_local=[0]).cpu().numpy()
```
Comment (duburcqa) [inline tests/test_rigid_physics.py]: No need to apply `.cpu().numpy()` if you use `assert_allclose`

**Comment by duburcqa** [comment]
Please rebase so we can merge.
